release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1055,6 +1074,9 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* The oc-mirror CLI plug-in cannot mirror {product-title} catalogs earlier than version 4.9.
+// TODO: Is there a BZ for this that we can list?
```

Comment:
@jpower432 Let me know if the wording for this 4.9 limitation sounds okay or not. Preview here: http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3567-rn/release_notes/ocp-4-11-release-notes.html#ocp-4-11-known-issues

Also, is there a BZ that we can link to for this?

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=2097210